### PR TITLE
add a simple demo watch command

### LINKED ISSUE #729: Create a `watch` command for demos

### PR DESCRIPTION
**Pull request by chee**
it doesn't do live-reloading, and it rebuilds everything (not just files that have changed) 
but it rebuilds the demos when you edit a file and serves the html on a port, so it's good enough for now maybe

### CODE REVIEW

**Comment by chee** [comment]
https://github.com/Financial-Times/origami/blob/5acab8fc88c94ae05dec0d8def2ae0b80ff3f02e/scripts/component/watch.bash

this is what we're talking about

File: components/o-message/origami.json [hunk 1]
```diff
@@ -284,7 +284,7 @@
 				"state": "pikachu",
 				"highlight": [
 					{
-						"copy": "Hurray",
+						"copy": "fuck",
```
Comment (akomiqaia) [inline components/o-message/origami.json:287]: Do we have to make this copy change? :D

Comment (chee) [reply components/o-message/origami.json:287]: woops looooooooool

File: scripts/component/watch.bash [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	echo goodbye
+	# kill all my children
+	pkill -P $$
+	# shut down the trap house
+	trap - INT
+	# die
+	exit
+}
+
+trap die INT
```
Comment (notlee) [inline scripts/component/watch.bash:23]: cool, TIL

File: tools/demo-build/Untitled-1.bat [hunk 3]
```diff
@@ -1,1006 +0,0 @@
-#!/bin/bash
-set -e
```
Comment (notlee) [inline tools/demo-build/Untitled-1.bat:2]: 👍 😬

**Comment by notlee** [comment]
Most helpful, thanks @chee! 🎉 Looks good to me. Would you mind updating the readme with the new requitements `rg entr` and a little section on the watch command?

**Review by notlee — APPROVED** [review]
excellent!

**Pull request merged**